MetaMask/metamask-mobile
# [Browser] Support downloading Apple Wallet passes on iOS

**Description**
Follow-up to https://github.com/MetaMask/metamask-mobile/pull/4052

This PR adds support for Apple Wallet(`.pkpass`) files using the `Linking` API instead of downloading the file as a blob. The current flow does not allow you to install the pass after it's been downloaded. 

**Screenshots/Recordings**
### Current Flow
![current](https://user-images.githubusercontent.com/3741055/174652001-7eb6cd78-f3eb-4632-99d6-3bda39681ae6.gif)

### Updated Flow
![pr](https://user-images.githubusercontent.com/3741055/174652007-78f331e0-b031-4ddf-bf88-51729b8d6c4a.gif)



## QA
- Test pkpass file: https://api.ethpass.xyz/api/public/download/GeneratedPass-0.09071582559205238.pkpass
- click the link, it should open the link in the browser and then prompt the user to add the pkpass file to their apple wallet
- downloading any other file should prompt the user to dowload
